redash/query_runner/elasticsearch.py

```diff
@@ -148,7 +148,11 @@ def parse_doc(doc, path=None):
             '''
             path = path or []
             result = []
-            for field, description in doc['properties'].items():
+            # index mappings contains "_default_" do not have "properties"
+            properties = doc.get('properties')
+            if not properties:
+                return result
+            for field, description in properties.items():
```

Comment:
Hi! What about making it a bit shorter and just writing this? 

`for field, description in doc.get('properties', {}).items():`

Thus, there will be no need for previous new lines

Reply:
Making python code short is a style thing, I am OK with that.
Besides, as latter comments pointed out, the `doc` may not be a dict. So maybe we still need code below to avoid the "not dict" issue.
```
if not isinstance(doc, dict):
    return result
```

Reply:
Or
```           
if not isinstance(doc, dict) or 'properties' not in doc:
    return result
```
and keep original code intact.